openedx/stanford/lms/envs/common.py

```diff
@@ -114,6 +114,7 @@
 # The following fields are available in the URL: {course_id} {student_id}
 PROGRESS_SUCCESS_BUTTON_URL = 'http://<domain>/<path>/{course_id}'
 PROGRESS_SUCCESS_BUTTON_TEXT_OVERRIDE = None
+SEARCH_FILTER_GENERATOR = 'openedx.stanford.lms.lib.courseware_search.lms_filter_generator.TileSearchFilterGenerator'
```

Comment:
Yeah, I'm okay with leaving this here for now;
we can move to JSON when the need arises.